src/components/Routes/Routes.js

```diff
@@ -35,7 +40,20 @@ export function Routes() {
       );
     }
     return (
-      <Route key={path[0]} path={path[0]} exact component={path[1].component} />
+      <Route key={path[0]} path={path[0]} exact>
+        <Login>
+          <Component />
+        </Login>
+      </Route>
     );
   });
+
+  return (
+    <Switch>
+      {routes}
+      <Route>
+        <NotFound path="*" />
```

Comment:
the `path` prop here is not being used, did you mean to put this on the `Route`?

Reply:
I meant to remove it, now removed.